models/_sector/dex/trades/celo/dex_celo_base_trades.sql

```diff
@@ -8,6 +8,10 @@
 {% set base_models = [
     ref('uniswap_v3_celo_base_trades'),
     ref('mento_v1_celo_base_trades'),
+    ref('mento_v2_celo_base_trades'),
```

Comment:
accidentally added mento_v2 twice?

Reply:
merge conflict resolving gone not exactly according to plan.. will fix! thanks for keeping a keen eye 🙏